components/director/pkg/customerrors/consts.go

```diff
@@ -0,0 +1,27 @@
+package customerrors
+
+//go:generate stringer -type ErrorType
+type ErrorType int
+
+const (
+	UnhandledError      ErrorType = 10
+	InternalError       ErrorType = 11
+	NotFound            ErrorType = 20
+	NotUnique           ErrorType = 21
+	InvalidData         ErrorType = 22
+	InsufficientScopes  ErrorType = 23
+	ConstraintViolation ErrorType = 24
+	TenantNotFound      ErrorType = 25
+	TenantNotExist      ErrorType = 26
```

Comment:
There has been a small update to the tenant errors, please rename it to `TenantIsRequired`